src/shared/data/contacts.js

```diff
@@ -0,0 +1,6 @@
+const contacts = {
```

Comment:
I believe it's better to have "data" files next to the code that uses it. In previous projects we used `.data.js` next to the code and ignore it for tests/coverage.

This way, it's easier to find static data about the code you're working on. Also, using a `data` folder means that if a project needs a lot of static data files, this folder will have a lot of files which hardens when searching for one specific.

Reply:
Agree

Reply:
I have a different opinion on that. Imagine that for some reason someone not really into the project wants to change some static data, like changing a date, an email or a phone number. I think that a file with data is easier to understand and manage, without needing to know our file structure and figuring where the information actually is.

Reply:
-- open issue